src/App.js

```diff
@@ -1,14 +1,30 @@
 import React from 'react';
-import { BrowserRouter as Router } from 'react-router-dom';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
```

Comment:
Hi @Kumilachew-g 👋 ,
- Good job so far.
- No linters errors
- You also followed gitflows, very nice.
- You may want to consider changing "Switch" to "Routes" because this is the latest version of "Switch" to avoid malfunctioning that may occur later.